junit-platform-console/junit-platform-console.gradle

```diff
@@ -33,18 +35,26 @@ idea {
 	}
 }
 
-mainClassName = "org.junit.platform.console.ConsoleLauncher"
+jar {
+	manifest {
+		attributes(
+			'Main-Class': 'org.junit.platform.console.ConsoleLauncher',
+			// generate test engine version information in single shared manifest file
+			// pattern of key and value: `'Engine-Version-{YourTestEngine#getId()}': '47.11'`
+			'Engine-Version-junit-jupiter': "$rootProject.version",
+			'Engine-Version-junit-vintage': "$vintageVersion"
+		)
+	}
+}
 
 javadoc {
 	classpath = project.sourceSets.main.compileClasspath + configurations.shadowed
 }
 
-jar.enabled = false
-
 shadowJar {
-	classifier = null
+	classifier = 'standalone'
```

Comment:
Exactly! In case it's easier to create a "completely new" artifact instead of just using a different qualifier, you may call it `junit-platform-console-standalone-1.0.0-SNAPSHOT.jar`.